lib/Doctrine/Query.php

```diff
@@ -1515,7 +1515,7 @@ public function getLimitSubquery()
         // add driver specific limit clause
         $subquery = $this->_conn->modifyLimitSubquery($table, $subquery, $this->_sqlParts['limit'], $this->_sqlParts['offset']);
 
-        $parts = $this->_tokenizer->quoteExplode($subquery, ' ', "'", "'");
```

Comment:
`quoteExplode` method of `Doctrine_Query_Tokenizer` (only place it's defined) does not take more than two parameters (and doesn't use func_get_args or similar).